pom.xml

```diff
@@ -53,6 +53,8 @@
         <dependency.check.version>6.1.6</dependency.check.version>
         <confluent.maven.repo>http://packages.confluent.io/maven/</confluent.maven.repo>
         <commons.codec.version>1.15</commons.codec.version>
+        <snakeyaml.version>1.28</snakeyaml.version>
```

Comment:
why do we need this? there is already a pin to 1.27 [here](https://github.com/confluentinc/kafka-connect-elasticsearch/blob/d0233415380c959e1e690ee7cc4e04a89faab44e/pom.xml#L109).

Reply:
Made the changes, thanks for pointing out.